packages/webpack/src/buildtime/emitSes.js

```diff
@@ -5,30 +5,28 @@ const {
 } = require('webpack')
 
 const lockdownSource = readFileSync(
-  path.join(require.resolve('ses'), '../lockdown.umd.min.js'),
+  path.join(require.resolve('ses'), '../lockdown.umd.js'),
```

Comment:
```suggestion
  require.resolve('ses'),
```

I'm pretty sure that's all that's necessary, because all of the distfiles are the same.

Reply:
except for the min one I used initially :D